examples/minimal_wp/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "minimal_wp"
+version = "0.1.0"
+authors = ["Denis Kolodin <deniskolodin@gmail.com>"]
+edition = "2018"
+
+[dependencies]
+yew = { path = "../../yew" }
+wasm-bindgen = "0.2"
+
+[lib]
```

Comment:
Can you please add a comment here explaining that `wasm-pack` builds crates as libraries and that's why this is needed?